client/state/selectors/get-post-share-scheduled-actions.js

```diff
@@ -9,23 +9,9 @@ import { moment } from 'i18n-calypso';
  */
 import { getPublicizeConnection } from 'state/selectors/';
 
-/**
- * Return a sharing-actions array propagaring data from publicize connections,
- * creating a Date (moment) instance for the shareDate, etc.
- *
- * @param {Object} state Global state tree
- * @param {Number} siteId Site ID
- * @param {Number} postId Post ID
- * @return {Object} sharing actions array
- */
-export default function getSitePostPublicizeSharingActions( state, siteId, postId ) {
-	const rawActions = get( state, [ 'sharing', 'publicize', 'sharePostActions', 'items', siteId, postId ], null );
-
-	if ( ! rawActions ) {
-		return null;
-	}
 
-	return map( rawActions, ( {
+export function mergeShareActionWithPublicizeConnections( state, postShareActions ) {
```

Comment:
Isn't there a file where we could put helper selectors like this? I mean, this helper selector is shared between `getPostShareScheduledActions` and `getPostSharePublishedActions` but this way, we put it into one of them which doesn't seem proper to me.